File: BBIS.Api/ClientApp/src/models/Inventory/InventoryGroupDto.ts
import { InventoryItemViewDto } from "./InventoryItemViewDto";

export interface InventoryGroupDto {
    BloodType: string;
    BloodRh: string;
    TotalNumberOfUnits: number;
    TotalExpiredCount: number;
    InventoryItems: Array<InventoryItemViewDto>;
}

File: BBIS.Application/DTOs/Inventory/InventoryGroupDto.cs
namespace BBIS.Application.DTOs.Inventory
{
    public class InventoryGroupDto
    {
        public string BloodType { get; set; }
        public string BloodRh { get; set; }
        public double TotalNumberOfUnits { get; set; }
        public int TotalExpiredCount { get; set; }

        public List<InventoryViewDto> InventoryItems { get; set; }
    }
}
